### PR TITLE
Add an option to control whether the title of the button is displayed #26

### PR DESCRIPTION
**Pull request by ettingshausen**
> I added the option.
> Using like this:
> 
> ```js-jsx
> <Fab
>             mainButtonStyles={{ backgroundColor: '#304FFE' }}
>             position={{ bottom: 0, right: 0 }}
>             icon={<MdAdd />}
>             event='click'
>             key='1'
>             showTitle={true}
>         >
> ```
> 
> gifhome_360x566
> 
> To keep the original look, just remove the `showTitle` prop.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ interface FabProps {
   icon: ReactElement;
   event?: 'hover' | 'click';
   children?: ReactElement[] | ReactElement;
+  showTitle: boolean;
```
Comment (dericgw) [inline index.d.ts]: A couple of things here:
1. The name `showTitle` implies that this must be set to `true` in order for the title to show, but what we really mean is "show the title without having to hover". Maybe a better name would be `alwaysShowTitle` or `persistTitle`?
2. This needs to be an optional prop `showTitle? : boolean`. We don't want to force the definition of a prop when we don't have to.

File: src/index.jsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```
Comment (dericgw) [inline src/index.jsx]: You can remove the `eslint-disable`. I am okay with the line break here. Prettier does a good job of keeping things tidy.

File: index.d.ts (hunk 1 above)
Comment (ettingshausen) [reply index.d.ts]: - `alwaysShowTitle` is fine
- I'll remove the `showTitle?` prop definition

File: src/index.jsx (hunk 2 above)
Comment (ettingshausen) [reply src/index.jsx]: I didn't notice that :joy:  
I'll remove this line.

**Comment by PeterKey** [comment]
Quite interested in this feature! Any news on this? Can I help out?

**Comment by dericgw** [comment]
@PeterKey I'm sorry for the delay. I'm going to release 3.0 this weekend and it will have this feature included.

**Comment by PeterKey** [comment]
No worries! Thank you so much, keep up the good work!

**Pull request merged**